src/libraries/System.IO.FileSystem/tests/Base/SymbolicLinks/BaseSymbolicLinks.FileSystemInfo.cs

```diff
@@ -0,0 +1,295 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public abstract class BaseSymbolicLinks_FileSystemInfo : BaseSymbolicLinks_FileSystem
+    {
+        protected abstract FileSystemInfo CreateFileSystemInfo(string path);
```

Comment:
```suggestion
        protected abstract FileSystemInfo GetLinkInfo(string path);
```

In order to avoid confussion with the other `Create*` methods in these tests, do not use "Create" for methods that don't delegate to `Create[As]*` APIs.

Reply:
I'll rename it GetFileSystemInfo, because not all these *Infos are for links, they can be for the targets.